api/src/test/java/uk/gov/dhsc/htbhf/claimant/entity/ClaimantTest.java

```diff
@@ -88,16 +88,6 @@ void shouldAlwaysReturnAnIdFromGetId() {
         Assertions.assertThat(id).isNotNull();
     }
 
-    @Test
-    void shouldReturnTheSameIdIfOneIsSet() {
```

Comment:
Id is no longer available to the builder and this test isn't really needed as it was just testing a property being set. This id test wasn't done for any other entity.

Reply:
as long as we have such a test for BaseEntity - it would be good to ensure that our id is always present, and remains the same.

Reply:
I've put this test back in.